projects/ziti-console-lib/src/lib/features/visualizer/network-visualizer/network-visualizer.component.scss

```diff
@@ -216,8 +217,8 @@ select:focus {
 
 @supports ((-webkit-backdrop-filter: saturate(110%) blur(15px)) or (backdrop-filter: saturate(110%) blur(15px))) {
     .nf-topology-filter-input {
-        background: var(--glass) !important;
-        -webkit-backdrop-filter: saturate(110%) blur(15px);
+        background: #ffffff80 !important;
```

Comment:
This should probably remain as a css variable. You can add application level css variables to the `styles.scss` stylesheet.

Reply:
will address this issue in next PR

Reply:
resolved with latest commit